pkgs/development/interpreters/python/cpython/2.7/default.nix

```diff
@@ -60,6 +60,24 @@ let
 
       ./properly-detect-curses.patch
 
+      # Fix python bug #27177 (https://bugs.python.org/issue27177)
+      # The issue is that `match.group` only recognizes python integers
+      # instead of everything taht has `__index__`.
+      # This bug was fixed upstream, but not backported to 2.7
```

Comment:
Do any of the other python versions (3.4, 3.5, ..) in-tree need this?

Reply:
Yes, everything <= 3.6 should have this issue. I only applied the patch for 2.7 because I only need it for 2.7 (for sage) and I wanted to minimize rebuilds. Since no 3.4/3.5 software seems to need it, I don't think its strictly necessary. Should I add it to 3.4 and 3.5 anyways?